README.md

```diff
@@ -68,96 +65,128 @@ you may receive this error:
+    Run ``ifconfig eth1`` while ssh'ed into the machine. The
+    appropriate IP is the ``inet addr:`` value, and the LMS server runs
+    on port ``8000`` by default. The value for ``EDXORG_BASE_URL`` should be
+    something like ``http://192.168.33.10:8000``
+- ``EDXORG_CLIENT_ID`` and ``EDXORG_CLIENT_SECRET``
 
-Open an iPython shell and execute these commands:
+    These values can be found in Django admin OAuth client section discussed above.
+    **Client id:** and **Client secret:** values should be auto-generated for
+    your new client. Use those values for the corresponding ``EDXORG_``
+    variables in the ``.env`` file
 
-      edxapp@precise64:~/edx-platform$ python manage.py lms --settings=devstack shell
-      Python 2.7.10 (default, Jun 29 2015, 22:38:23)
-      Type "copyright", "credits" or "license" for more information.
+## Docker Container Configuration and Start-up
 
-      IPython 4.1.2 -- An enhanced Interactive Python.
-      ?         -> Introduction and overview of IPython's features.
-      %quickref -> Quick reference.
-      help      -> Python's own help system.
-      object?   -> Details about 'object', use 'object??' for extra details.
+First, create a ``.env`` file from the sample in the codebases:
 
-      In [1]: from django.contrib.auth.models import User
+    cp .env.sample .env
 
-      In [2]: User.objects.get(username='staff')
-      Out[2]: <User: staff>
+Set the ``EDXORG_BASE_URL``, ``EDXORG_CLIENT_ID``, and ``EDXORG_CLIENT_SECRET``
+variables in the ``.env`` file appropriately.
 
-      In [3]: staff=_
+For first-time container start-up, start it with a full build:
 
-      In [4]: staff.is_superuser=True
+    docker-compose up --build
 
-      In [5]: staff.save()
+In another terminal tab, navigate the the Micromasters directory
+and add a superuser in the now-running Docker container:
 
-      In [6]:
-      Do you really want to exit ([y]/n)? y
+    docker-compose run web python manage.py createsuperuser
 
-Start the LMS:
+Starting the container after this can be done without the ``--build``
+param: ``docker-compose up``
 
-    paver devstack lms
+_OSX Users: The running server should be accessible at
+http://192.168.99.100:8079_
 
-Navigate to ``localhost:8000/admin`` and login with the superuser account.
-Scroll down to the "Oauth2" section and open "Clients".
-Create a new client
-    name - your choice of name, mine is ``micromasters-local``.
-    Url - the URL of your local Micromasters instance, a URL on port ``8087``.
-    Redirect url: the same URL, but with ``/complete/edxorg/`` at the end. 
-    Client type: Chose "Confidential (Web applications)"
+Note: Your IP address may vary depending on what address Docker 
+assigns to your container.  If the IP address above doesn't work, enter 
+this command:
 
-Make a note of the Client id and the Client secret; you need them to
-configure Micromasters.  Don't forget to save your new client.
+    docker-machine ls
```

Comment:
this is another solution to my [earlier comment](https://github.com/mitodl/micromasters/pull/313/files#r63185965)